guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -70,120 +60,209 @@ If you lose connection to the command shell where the upgrade command is running
 enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
 ----
 +
-. In the configuration file, replace `My_Organization` in the `baseurl` with the correct organization label.
-To obtain the organization label, enter the command:
+* In the configuration file, replace 
+** `/etc/pki/katello/certs/org-debug-cert.pem` in `sslclientcert` and `sslclientkey` with the location of the downloaded organization debug certificate.
+** `satellite.example.com` with correct FQDN 
+** `My_Organization` with the correct organization label in the `baseurl`. To obtain the organization label, enter the command:
++
+----
+# hammer organization list
+----
+
+* Run the `reposync` command as follows:
 +
+[options="nowrap" subs="attributes"]
 ----
-# ls /var/lib/pulp/published/yum/https/repos/
+# cd ~
+# reposync --delete --download-metadata -p satellite_repos -n \
+ -r {RepoRHEL7ServerAnsible} \
+ -r {RepoRHEL7Server} \
+ -r {RepoRHEL7ServerSatelliteServerProductVersion} \
+ -r {RepoRHEL7ServerSatelliteMaintenanceProductVersion} \
+ -r {RepoRHEL7ServerSoftwareCollections}
 ----
+
+* This will download the contents of these repositories from the Upstream Satellite and store it in the directory `~/satellite-repos`.
+* `reposync` in RHEL 7 at this point does not download the yum metadata. It downloads just the rpms. Due to we would need to manually run `createrepo .` in each sub-directory of `satellite-repos`. You may use the following to automate this. 
 +
+[options="nowrap" subs="attributes"]
+----
+# cd ~/satellite-repos
+# for directory in $(ls -d */)
+do
+  echo "Processing $directory"
+  cd $directory
+  createrepo .
+  cd ..
+done
+----
 
-. Ensure that the `rubygem-foreman_maintain` package that provides `{foreman-maintain}` is installed and up to date:
+* Verify that the rpm's have been downloaded and the repodata directory is generated in each of the subdirectories of `~/satellite-repos`.
+* Archive the contents of the directory 
 +
-[options="nowrap"]
+[options="nowrap" subs="attributes"]
 ----
-# yum install rubygem-foreman_maintain
+# cd ~
+# tar czf satellite-repos.tgz satellite-repos
 ----
 
-. Check the available versions to confirm the next minor version is listed:
+* We can now use the generated `satellite-repos.tgz` file to upgrade in the Downstream {ProjectServer}.
+
+*On the Downstream {ProjectServer}*
+
+* Copy the generated `satellite-repos.tgz` file from above to Downstream {ProjectServer}.
+* Extract the archive to anywhere accessible by root. In the example below we are extracting it at `/root`
 +
-[options="nowrap" subs="+quotes,attributes"]
+[options="nowrap" subs="attributes"]
+----
+# cd /root
+# tar zxf satellite-repos.tgz
+----
+* Create a new yum configuration file as follows:
++
+[options="nowrap" subs="attributes"]
+----
+# vi /etc/yum.repos.d/redhat-local.repo
```

Comment:
```suggestion
# vi /etc/yum.repos.d/redhat-local.repo
----
+
```

Reply:
Alternative: don't suggest an editor to the user?